doc/api/stream.md

```diff
@@ -265,7 +270,9 @@ changes:
   * `error` {boolean|undefined}
   * `readable` {boolean|undefined}
   * `writable` {boolean|undefined}
-  * `signal`: {AbortSignal|undefined}
+  * `signal` {AbortSignal|undefined}
+  * `cleanup` {boolean|undefined} If `true`, removes the listeners registered by
+    this function before the promise is fulfilled. **Default:** `false`.
```

Comment:
Don't think that the convention for options parameter types is to include `undefined`?

Reply:
IMO that's an issue for a different PR because this PR follows the same convention as the rest of the parameters.